packages/labs/gen-wrapper-vue/goldens/test-element-a/vite.config.ts

```diff
@@ -0,0 +1,34 @@
+import vue from '@vitejs/plugin-vue';
+import typescript from '@rollup/plugin-typescript';
+
+// https://vitejs.dev/config/
+export default {
+  build: {
+    rollupOptions: {
+      external: () => true,
+      input: ['./src/element-a.ts'],
+      preserveModules: true,
+      preserveEntrySignatures: true,
+      output: [
```

Comment:
If you're outputting both UMD & ESM, do you need to specify different entrypoints in package.json?

Reply:
Removed for now to avoid dealing with the complexity.  Can add back if we decide we want to support this.